frontend/app/src/form/schema.tsx

```diff
@@ -51,6 +64,27 @@ const estimated_time = estimation.map((e) => {
   };
 });
 
+const helpType_wanted_coding_task_schema = help_wanted_coding_task_schema.map((e) => {
+  return {
+    value: e,
+    label: e
+  };
+});
+
+const helpType_wanted_other_schema = help_wanted_other_schema.map((e) => {
+  return {
+    value: e,
+    label: e
+  };
+});
+
+const estimated_budget_15_min_options = estimated_budget_15_min.map((e) => {
+  return {
+    value: e,
+    label: e
+  };
+});
+
```

Comment:
Seems like there's a bunch of repeated code here is there any way to do this more simply?